src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public float getArea();
```

Comment:
we don't have to write `public` here (it is default modifier for methods in interfaces)